augly/image/intensity.py

```diff
@@ -42,6 +42,8 @@ def blur_intensity(radius: int, **kwargs) -> float:
 def brightness_intensity(factor: float, **kwargs) -> float:
     return mult_factor_intensity_helper(factor)
 
+def skew_intensity(level: float, **kwargs):
+    return min(level, 100.0)
```

Comment:
couple things:

1. does this mean that the maximum **reasonable** value for `level` should be 100? this function is supposed to return the intensity of the `skew` applied given the `level` parameter. if `level` is supposed to be a value from 0-1 then the intensity should be `level * 100`
2. please normalize the output of this function to be a value from 0-100. currently, if i input a negative value for level, I'll get a negative intensity. 
3. Please insert the intensity function in alphabetical order with respect to the other functions already in this file

Reply:
1. No a suitable maximum range of level parameter is 2. Also I've commented below about outputs of different value of level parameter.
2. Also we can provide the negative value for level parameter the basic difference is that it will skew image in opposite direction. Normalization needs there when we multiply level with 100 or something but here we are going to take the minimum value for intensity function.
3. Yes, I've.